submissions/Quickstart_Guide.md

```diff
@@ -0,0 +1,104 @@
+# Quickstart Guide to Integrate Chimoney API
+
+Welcome to the Chimoney Quickstart Guide! This guide will help you easily integrate the Chimoney API into your applications.
+
+## Step 1: API Key Generation
+
+To start using the Chimoney API, you need to sign up and generate an API key from the developer dashboard:
+
+1. Go to the [Chimoney Developer Dashboard](https://dash.chimoney.io/auth/signin?next=/).
+2. Sign up for an account or log in if you already have one.
+3. Navigate to the "API Keys" section.
+4. Click on "Generate New API Key" and save your key securely.
+
+## Step 2: SDK Installation
+
+Chimoney provides SDKs for popular programming languages to simplify integration. Below are instructions for installing the SDK for Node.js and Python.
+
+### Node.js
+
+To install the Chimoney SDK for Node.js, use npm:
+
+```bash
+npm install chimoney-sdk
+```
+
+### Python
+
+To install the Chimoney SDK for Python, use pip:
+
+```bash
+pip install chimoney-sdk
+```
+
+## Step 3: Making a Sample Request
+
+Here’s how to make a basic API call to initiate a payout using the Chimoney SDK.
+
+### Node.js Example
+
+```javascript
+const Chimoney = require('chimoney-sdk');
+
+const chimoney = new Chimoney('YOUR_API_KEY');
+
+chimoney.payout({
+    amount: 100,
+    currency: 'USD',
+    recipient: 'recipient@example.com',
+})
+.then(response => {
+    console.log('Payout successful:', response);
+})
+.catch(error => {
+    console.error('Error initiating payout:', error);
+});
+```
+
+### Python Example
+
+```python
+from chimoney import Chimoney
+
+chimoney = Chimoney(api_key='YOUR_API_KEY')
+
+response = chimoney.payout(amount=100, currency='USD', recipient='recipient@example.com')
+
+print('Payout successful:', response)
+```
+
+## Step 4: Overview of Key Features
+
+The Chimoney API provides several key functionalities, including:
```

Comment:
Amazing article @Brijeshthummar02!

Just a few changes:

- Please remove the last 2 on this list i.e `Balance retrieval` and `Transaction History` and replace them with: 
  - Multi-currency wallets
  - Payment Requests
  - Redemption of value
Also, add detailed descriptions for each functionality.

For example for `Payouts` a sample well-detailed description can be: 

> With Chimoney’s API, you can initiate seamless payouts to over 130 countries. Whether it's to banks, mobile money, airtime, or even Interledger-enabled accounts.